freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/create/FreeIpaRecommendationResponse.java

```diff
@@ -0,0 +1,51 @@
+package com.sequenceiq.freeipa.api.v1.freeipa.stack.model.create;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import io.swagger.annotations.ApiModel;
+
+@ApiModel("FreeIpaRecommendationV1Response")
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class FreeIpaRecommendationResponse {
```

Comment:
Is there a clear recommendation about this? I am asking because i have checked the nearby response objects during implementation and I couldnt find a clear pattern about this.

Reply:
Not anything that would be enforced. The rest of the response types here (e.g. `VmTypeResponse`) all implement `JsonEntity`, which is also `Serializable`. Jackson likely does not mandate this, though, so up to you if this makes sense.